nixos/modules/services/networking/networkmanager.nix

```diff
@@ -505,7 +505,7 @@ in {
     # TODO: keep an eye on modem-manager releases as this will eventually be added to the upstream unit
     systemd.services.ModemManager.serviceConfig.ExecStart = [
       ""
-      "${pkgs.modemmanager}/sbin/ModemManager --filter-policy=STRICT"
```

Comment:
Actually, we should just get rid of this option definition altogether.

Reply:
Oh yeah, no point in overriding things anymore. I'll fix that up.